FilterPlugin/src/main/java/io/antmedia/filter/FilterAdaptor.java

```diff
@@ -174,7 +174,7 @@ public boolean update()
 
 			String videoFilterArgs = "video_size="+videoStreamParams.codecParameters.width()+"x"+videoStreamParams.codecParameters.height()+":"
 					+ "pix_fmt="+videoStreamParams.codecParameters.format()+":"
-					+ "time_base="+videoStreamParams.timeBase.num()+"/"+videoStreamParams.timeBase.den()+":"
+					+ "time_base="+1+"/"+1000+":"
```

Comment:
Do we have always 1/1000 for every streams(RTSP, WebRTC, RTMP)?

Reply:
It works well for RTMP and WebRTC. For RTSP there is another issue: https://github.com/ant-media/Ant-Media-Server/issues/3878